src/classes/TestReminderProcessor.cls

```diff
@@ -0,0 +1,21 @@
+@isTest
+private class TestReminderProcessor {
+    
+    @TestSetup
+     static void loadData(){       
+        List<Invoice__c> inv = new List<Invoice__c>();
+        inv = TestDataClass.loadInvoices();
+         inv.get(0).Draft__c = false;
+         update inv;
+    }
+    
+    @isTest
+     static void testReminderProcessor() {        
```

Comment:
W teście nie sprawdzamy ile rekordów było przed uruchomieniem batch'a ponadto myślę że warto użyć System.runAs() w kontekście system admina. Nie zawsze to admin odpala testy i może wystapić przypadek kiedy np. deploment user ma inny profil i ten test wtedy się wywali.

Reply:
jak mam to odpalic w kontekscie system admina?

Reply:
System.runAs(tutaj user ktory ma profil System Admina - jesli nie mamy takiego w metodzie tworzącej dane testowe to trzeba go utworzyć).

Reply:
rozumiem wszystko, ale chyba nie miałes sie do czego przyczepic i juz wymyslasz xD